run_dir/design/application.html

```diff
@@ -11,14 +11,12 @@
 <div class="container-fluid">
   <h2 id="page_title">Projects with application <em>{{ application }}</em></h2><br>
   <ul id="project_list">
-		<tbody></tbody>
-  </table>
 </div>
 
 <script>
 $.getJSON("/api/v1/application/{{ application }}", function(data) {
   $.each(data, function(k, project) {
-    $("#project_list").append('<li><a href="/projects/' + project + '">' + project + '</a></li>')
+    $("#project_list").append('<li><a class="text-decoration-none" href="/projects/' + project + '">' + project + '</a></li>')
   })
 })
```

Comment:
I'm pretty sure this page is not used anywhere, but it seems to work now.

Reply:
I think it navigates to this page when you click on the links in the table in the Applications page.